### DISCUSSION TITLE
Buttons accidentally appeared on kitty's title-bar on Gnome/Wayland

### DISCUSSION THREAD

**Discussion by terry-daviass** [kovidgoyal/kitty#7524] [Q&A] [answered]
![image](https://github.com/kovidgoyal/kitty/assets/165209708/44ffeda7-df00-4a0f-a775-5898b02e0cbe)

Previously, my title bar only displayed a cat emoji and the word "kitty." However, a new style has been applied to my terminal, adding unwanted buttons to the title bar. I have not found any settings in kitty.conf responsible for this title bar style change. I would like to remove these buttons and identify all settings related to the title bar.
Any assistance would be greatly appreciated.

**Comment by kovidgoyal** [comment, marked as answer]
There are no settings related to the title bar. And the buttons are
there by design not accident.
